README.md

```diff
@@ -64,38 +74,45 @@ var handle = client.execute({
 handle.stream(function(err, data) { console.log(data); });
 ```
 
-Object `options` is an optional map and may contains following fields:
+The `options` object is an optional map and can contain following fields:
 
-- **signalflowEndpoint** - string, `wss://stream.us0.signalfx.com` by default. Override this if you are in a different realm than `us0`.
-- **apiEndpoint** - string, `https://api.us0.signalfx.com` by default. Override this if you are in a different realm than `us0`.
-- **webSocketErrorCallback** - function, Throws an Error event by default. Override this if you want to handle a websocket error differently.
+| Field                    | Type     | Description                                                               | Default                                                                                            |
+|--------------------------|----------|---------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------|
+| `signalflowEndpoint`     | string   | Sets the SignalFlow endpoint.                                             | `wss://stream.us0.signalfx.com`. If you aren't in `us0`, change the realm.                         |
+| `apiEndpoint`            | string   | Sets the API endpoint.                                                    | `https://api.us0.signalfx.com`. If you aren't in `us0`, change the realm.                          |
+| `webSocketErrorCallback` | function | Sets the callback function that is invoked when a WebSocket error occurs. | Throws an error event. If you want to handle the WebSocket error differently, change the behavior. |
```

Comment:
@seemk Are these descriptions OK?

Reply:
I think the `Throws an error event` is a bit confusing. By default the logic is to throw an error, but you can override this if you set this callback so instead of an error being thrown the callback is invoked.